.vscode/settings.json

```diff
@@ -1,4 +1,24 @@
 {
-    "jest.pathToJest": "npm run test -- --coverage",
-    "editor.formatOnSave": true
-}
\ No newline at end of file
+  "jest.pathToJest": "npm run test -- --coverage",
+  "editor.formatOnSave": true,
```

Comment:
since we're disabling this on the main file types, should we just go ahead and turn this off?

Reply:
Ummm I don't think so. The goal of all the config is to have only 1 autoformatter run. So the `eslint.autoFixOnSave` will run and fix the kinds that have `editor.formatOnSave` turned off.

Reply:
we still want `editor.formatOnSave` on, for instance, `css` or `json` files